modules/scala/scala-interpreter/src/main/scala/almond/ScalaInterpreter.scala

```diff
@@ -288,8 +288,24 @@ final class ScalaInterpreter(
         else
           predef
 
-      val stmts = Parsers.split(code).get.get.value
-      ammInterp0.processLine(code, stmts, -1, silent = true, () => ())
+      def handlePredef(): Unit = {
+        val stmts = Parsers.split(code).get.get.value
+        val predefRes = ammInterp0.processLine(code, stmts, 9999999, silent = false, () => ())
```

Comment:
This `999` may have fixed a bug I stumbled upon wherein I passed a multi-line predef like:
```
import foo
foo.init()
```

Which generated console output:
```
cmd$minus1.sc:94: Invalid literal number
val res-1_2 = init()
```

Reply:
It should solve that, yes.